source/Plugins/Language/Swift/SwiftOptional.cpp

```diff
@@ -42,7 +42,6 @@ std::string lldb_private::formatters::swift::SwiftOptionalSummaryProvider::
 // retrieve the value of the Some case..
 static PointerOrSP
 ExtractSomeIfAny(ValueObject *optional,
-                 lldb::DynamicValueType dynamic_value = lldb::eNoDynamicValues,
                  bool synthetic_value = false) {
```

Comment:
All the callers of `ExtractSomeIfAny`, always pass `true` to `synthetic_value`. Can we get rid of the extra argument?

Reply:
Yes, I have that cleanup (and another for IsNonTriviallyManagedReferenceType) ready locally, but I'd rather do it in a different PR.